client/src/containers/VerifyEmailPage.tsx

```diff
@@ -90,11 +109,36 @@ const VerifyEmailPage = withI18n()((props: withI18nProps) => {
     </div>
   );
 
+  const alreadyVerifiedPage = () => (
+    <div className="text-center">
+      <Trans render="h4">Your email is already verified</Trans>
+      <br />
+      <Trans className="text-center">You will be redirected back to Who Owns What in:</Trans>
+      <br />
+      <br>{updateCountdown()}</br>
+      <Trans className="d-flex justify-content-center">
+        <span id="countdown">{delaySeconds}</span> seconds
+      </Trans>
+      <br />
+      <br />
+      <Trans className="text-center">If you are not redirected, please click this link:</Trans>
+      <br />
+      <a href={redirectUrl}>{redirectUrl}</a>
+    </div>
+  );
+
   return (
     <Page title={i18n._(t`Verify your email address`)}>
       <div className="TextPage Page">
         <div className="page-container">
-          {isVerified ? successPage() : isExpired && expiredLinkPage()}
+          {!loading &&
```

Comment:
`loading` var is necessary to prevent flickering of a different state as the AuthClient async calls are getting resolved. 

the nested ternary operator is gross but i didn't have any other ideas. lmk if you do!